tests/integration/test_monitoring_stack.py

```
@@ -0,0 +1,265 @@
+    # 1. Send a test trace
+    otlp_exporter = OTLPSpanExporter(endpoint="localhost:4317", insecure=True)
+    trace.set_tracer_provider(TracerProvider())
+    tracer_provider = trace.get_tracer_provider()
+    tracer_provider.add_span_processor(BatchSpanProcessor(otlp_exporter))
+
+    tracer = trace.get_tracer("e2e-test")
+    test_trace_id = None
+
+    with tracer.start_as_current_span("e2e-test-span") as span:
+        span.set_attribute("test.e2e", "true")
+        test_trace_id = format(span.get_span_context().trace_id, "032x")
```

Ruff (0.14.4)

Local variable `test_trace_id` is assigned to but never used

Remove assignment to unused variable `test_trace_id`

(F841)